databricks/koalas/typedef.py

```diff
@@ -172,6 +170,18 @@ def _check_compatible(arg, sig_arg: X):
     assert False, (arg, sig_arg)
 
 
+def dict_sanitizer(input_dict):
+    """
+    This function checks if elements inside a given dict are supported by Spark.
+    :param input_dict: dict
+    :raises TypeError if any element is not supported by Spark
+    """
+    _possible_type_values = [int, str, float, np.float64, np.float, bool]
```

Comment:
Hm, actually I don't think we support np.float ones. Let me check and make a small followup